examples/append_conc_to_h5file.py

```diff
@@ -9,14 +9,17 @@
 'actual_conc'.
 """
 from pyspecdata import *
-import h5py,os
-data_info = dict({"filename":'220126_Ras_M67R1a_capProbe.h5', #file that is being edited
-        "file_location":"ODNP_NMR_comp/ODNP", #location of file
-        "actual_conc": 72e-6}
-        )#concentration of the dataset in M
```

Comment:
you should review my changes carefully, but be mindful that some of these come from running black